pkg/installation/installation.go

```diff
@@ -22,10 +22,11 @@ type Installation struct {
 	Provider          string
 	Codename          string
 	CACert            string
+	SourcePath        string
```

Comment:
Storing source path (the URL of Kubernetes API or another service provided to kgs as an argument). It is used to determine whether the Installation instance in memory can be re-used. In case an installation for a different source path is needed, it is reloaded from Athena.